addons/account/static/src/scss/section_and_note_backend.scss

```diff
@@ -2,24 +2,24 @@
 // The goal of this file is to contain CSS hacks related to allowing
 // section and note on sale order and invoice.
 
-table.o_section_and_note_list_view tr.o_data_row.o_is_line_note,
-table.o_section_and_note_list_view tr.o_data_row.o_is_line_note textarea[name="name"],
-div.oe_kanban_card.o_is_line_note {
+table.o_section_and_note_list_view tr.o_data_row.o_is_invl_note,
+table.o_section_and_note_list_view tr.o_data_row.o_is_invl_note textarea[name="name"],
+div.oe_kanban_card.o_is_invl_note {
```

Comment:
Take care this css is also used by sale order (not sure at 100%, to check) so I think you should keep the same class names